lib/src/over_react_test/dart_util.dart

```diff
@@ -0,0 +1,6 @@
+/// Returns whether the current runtime is `dartdevc`.
+bool runningInDDC() {
```

Comment:
This should be called `assertsEnabled`, not `runningInDDC`, since it's possible to enable asserts in dart2js. Same with the `ddcOnly` flag.

Also, perhaps instead of exposing this raw check, we should expose a more specific API that correlates better to how consumers will use it, like `runtimeSupportsPropTypeWarnings`

Reply:
@greglittlefield-wf I'm on board with renaming it - but IMO, the consumer shouldn't need something like `runtimeSupportsPropTypeWarnings`, since as a result of these changes they should just be able to run their tests - and if they run them in DDC, the propTypes will be checked, otherwise the `logsPropError` matcher(s) are essentially a "no-op" that will always result in a passing test.

Reply:
Then why are we exposing it?

Reply:
Fair point I guess... 

I mean - there are other uses (besides `propTypes`) for it IMO - but if you'd be more comfortable not exposing it, I'm fine with that, too @greglittlefield-wf.

Reply:
I'd vote not to expose it unless we have an immediate use-case in mind that's directly related to testing React components

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/371059c13071db43fbacddc8c53feb926614a29b